docs/advanced_topics/customisation/extending_draftail.rst

```diff
@@ -304,7 +304,7 @@ Finally, we register the JS components of our plugin:
 
 And that’s it! All of this setup will finally produce the following HTML on the site’s front-end:
 
-.. code-block:: html
+.. code-block:: html+django
```

Comment:
This block is only `html`, should I revert this ?

Reply:
Yes, we should only use `html+django` for template code - this is showing the output of the template, so `html` is correct.